packages/ocamlbuild/ocamlbuild.0.14.2/opam

```diff
@@ -26,8 +26,14 @@ build: [
     "OCAMLBUILD_MANDIR=%{man}%"
     "OCAML_NATIVE=%{ocaml:native}%"
     "OCAML_NATIVE_TOOLS=%{ocaml:native}%"
-  ]
-  [make "check-if-preinstalled" "all" "opam-install"]
+  ] {os != "win32"}
+  [make "check-if-preinstalled" "all" "opam-install"] {os != "win32"}
+  [make "all"] {os = "win32"}
+]
+install: [
+  [make "install"] { os = "win32" }
+  ["mkdir" "-p" "%{lib}%/ocamlbuild"] { os = "win32" }
+  ["install" "-m" "0644" "META" "%{lib}%/ocamlbuild"] { os = "win32" }
 ]
```

Comment:
I'm not sure if opam will realise on non-Windows that this package _doesn't_ have an install section (@rjbou?) which technically does impact non-Windows users.

Could we get away with adding an additional package for Windows (say `ocamlbuild.0.14.2+fdopen`) and marking this one as `available : os != "win32"`?

Reply:
The solver will select the +win variant anyway, but is it worth having `available: os != "win32"` in ocamlbuild.0.14.2/opam just for "belt and braces"?